ucsb-cs156-f23/proj-happycows-f23-5pm-4
# UI Improvements to Instructor Report Page

## Overview

In this PR, we make UI improvements to the Instructor Report page. We format dollar amounts as currency and all other numbers are right justified rather than left justified in their columns.

## Screenshots

<img width="1331" alt="Screenshot 2023-11-21 at 12 13 13 AM" src="https://github.com/ucsb-cs156-f23/proj-happycows-f23-5pm-4/assets/46696490/64d5db00-ce6e-4462-a8ca-d7e526be6258">



## Validation

1. Download this branch.
2. Run the project.
3. Go to the Manage Jobs page and generate an Instructor Report.
4. Go to the Instructor Report page and make sure the values are right justified and dollar amounts are formatted as currency.

## Tests


## Linked Issues

Closes #2
